subset/security/module_manifest.json

```diff
@@ -1,6 +1,12 @@
 {
-    "security.tls.v3" : {
-        "desc" : "Verify the device supports TLS 1.2 or above (as a client)"
+    "security.tls.v1_3" : {
+        "desc" : "Verify the device supports TLS 1.3 (as a client)"
```

Comment:
should be "description" -- no need to abbreviate here -- I don't think we have any schema enforcement for this yet...

Reply:
Updated